brainstormforce/ultimate-addons-for-gutenberg
# Fix: Block Conditions - Conflicting with the WPForms.

### Description

Trello - https://trello.com/c/qGR6uGBc/209-fix-block-conditions-conflicting-the-wpforms

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Create a simple WPForm.
- Add Form to the page using WPForms block and check if an error occurred or not.

### Checklist:
